### PR TITLE
fix: Bold by Devoteam

### PR DESCRIPTION
**Pull request by afonsojramos**
It seems that these companies updated their websites as the build was failing: https://github.com/marmelo/tech-companies-in-portugal/actions/runs/3476885472/jobs/5812508041

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 | [Accenture](https://www.accenture.com/pt-pt) [:rocket:](https://www.accenture.com/pt-pt/careers) | Software Development, Cyber Security. | `Lisboa` `Porto` <br> `Remote`|
 | [AddCode](https://addcode-io.breezy.hr) [:rocket:](https://addcode-io.breezy.hr/#positions) | Software engineering. | `Porto` |
 | [Adentis](https://www.adentis.pt/) [:rocket:](https://www.adentis.pt/en/work-with-us/job-list) | Outsourcing, customized solutions and R&D. | `Lisboa` |
-| [Armis](https://www.armisgroup.com/) [:rocket:](https://www.armisgroup.com/company/careers/) | Technology development and consultancy. | `Lisboa` `Porto` |
+| [Armis](https://www.armis.pt/) [:rocket:](https://www.armis.pt/empresa/carreiras/) | Technology development and consultancy. | `Lisboa` `Porto` |
```
Comment (vpereira01) [inline README.md:66]:
```suggestion
| [Armis](https://www.armisgroup.com/) [:rocket:](https://www.armisgroup.com/company/careers/) | Technology development and consultancy. | `Lisboa` `Porto` |
```

I think you can revert this change, it seemed like a temporary problem.

**Review by marmelo — APPROVED** [review]

**Review by vpereira01 — APPROVED** [review]
:+1:
